SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialShadowActor.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma optimize("", off)
+#include "EngineClasses/SpatialShadowActor.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialShadowActor);
+
+USpatialShadowActor::USpatialShadowActor(const FObjectInitializer& ObjectInitializer /*= FObjectInitializer::Get()*/)
+	: Super(ObjectInitializer)
+	, EntityId(SpatialConstants::INVALID_ENTITY_ID)
+	, ReplicatedPropertyHash("")
```

Comment:
A default constructor I believe would be sufficient?

Reply:
However as mentioned in header, an integer seems like a more appropriate choice of data type?